src/load.ts

```diff
@@ -109,34 +112,48 @@ function getMutation(config, argv) {
 }
 
 function findReturnExpression(mutationField) {
+    // console.log(chalk.yellow("At findReturnExpression, the mutationField is " + JSON.stringify(mutationField)))
     const returnType = getNamedType(mutationField.type)
     if (returnType instanceof GraphQLObjectType) {
        const fields = (returnType as GraphQLObjectType).getFields();
        const field = Object.keys(fields).find((x) => (getNamedType(fields[x].type) === GraphQLID)) || Object.keys(fields)[0]
-       return `{ ${field} }`
+       // console.log("field at findReturnExpression is: " + field)
+	return `{ ${field} }`
     }
     return "";
 }
 
 function buildMutations(mutationField, args, data, mapping,delim) {
-  const rMapping = {};
+
+
+  // const rMapping = {}; // unnecessary given direct mapping is used
   const regexp = new RegExp(delim + "\s*");
-  Object.keys(mapping).forEach((k) => rMapping[mapping[k]]=k);
+  // Object.keys(mapping).forEach((k) => rMapping[mapping[k]]=k); // unnecessary given direct mapping is used
+
   const mutations = 
   data.map((row,idx) => {
+
     var fullfilled = true;
     const params = Object.keys(args).map( (key) => { 
       const arg = args[key];
-      const column=(rMapping[key]||key).toString();
+
+
+      //  const column=(rMapping[key]||key).toString(); // original
+      //  console.log("Correcting the key mapping data with const column =  mapping[key].toString() is " + (mapping[key]).toString())
+      const column = mapping[key].toString()
```

Comment:
what happens if there is no entry for `key` in the mapping? Then the original name from the mutation args has to be used.